.gitlab-ci.yml

```diff
@@ -29,19 +29,17 @@ build:
 
 build_tagged_image:
   stage: deploy
-  only:
-    - tags@rpki/rpki-publication-server
   image: docker:latest
   services:
     - docker:dind
   script:
     - echo $CI_REGISTRY_PASSWORD | docker login -u $CI_REGISTRY_USER --password-stdin $CI_REGISTRY
-    # Abort if not on a tag
-    - if [ -z "${CI_COMMIT_TAG}" ]; then echo "Not releasing docker image tagged with a version."; exit 0; fi;
-    - echo "Releasing this image as :latest and :${CI_COMMIT_TAG}"
-    - docker build --tag $CI_REGISTRY_IMAGE:$CI_COMMIT_TAG --tag $CI_REGISTRY_IMAGE:latest .
-    - docker push $CI_REGISTRY_IMAGE:$CI_COMMIT_TAG
+    - echo "Releasing this image as :latest and :${CI_COMMIT_SHORT_SHA}"
+    - docker build --tag $CI_REGISTRY_IMAGE:$CI_COMMIT_SHORT_SHA --tag $CI_REGISTRY_IMAGE:latest .
```

Comment:
Since I've googled, might as well document it here: https://stackoverflow.com/a/35565384/397758

Reply:
Yes it's quite helpful to build multiple tags at once, especially since it reduced build time.

It's a shame that they never got around to fixing `push` for multiple tags, but docker/cli#267 is still open (three years and counting).